opl/generators/inventory_ingress.py

```diff
@@ -18,19 +18,20 @@ class InventoryIngressGenerator(opl.generators.generic.GenericGenerator):
 
     installed_packages = []
 
-    def __init__(self, count, fraction=1, relatives=100, addresses=3, packages=500, template='inventory_ingress_RHSM_template.json.j2'):   # noqa: E501
+    def __init__(self, count, fraction=1, relatives=100, ip_v4_addresses=10, ip_v6_addresses=10, mac_addresses=1, packages=500, template='inventory_ingress_RHSM_template.json.j2'):   # noqa: E501
```

Comment:
I would just keep current API we have as we might use it in some tests:
```
[jhutar@jhutar 2023-01-10T09:37:23 iperf]$ git grep 'inventory_ingress\..*('
host_synchronizer/script-populate.py:    payload_generator = opl.generators.inventory_ingress.PayloadRHSMGenerator(
inventory/script-generate.py:    payload_generator = opl.generators.inventory_ingress.PayloadRHSMGenerator(
xjoin/diff_sys_profile.py:    generator = opl.generators.inventory_ingress.PayloadRHSMGenerator(count=count, template='inventory_ingress_puptoo_template.json.j2')
yupana/script-generate.py:            host_generator = opl.generators.inventory_ingress.PayloadRHSMGenerator(
```
To make it simple, I would just use addresses option to set number of all 3 addresses (IPv4, IPv6 and MAC).